<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>681</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bAsiakas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Asiakas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bKortti">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Kortti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bTapahtumat">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tapahtumat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tervetuloa!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bSaldo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bOtto">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Otto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bDebit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Debit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bCredit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deppiitti vai creddiitti</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
